node/src/sub_lib/neighborhood.rs

```diff
@@ -162,6 +163,7 @@ pub struct NodeDescriptor {
     pub blockchain: Chain,
     pub encryption_public_key: PublicKey,
     pub node_addr_opt: Option<NodeAddr>,
+    pub country_code: String,
```

Comment:
Still not sure why it belongs here. Hopefully you are gonna come to discuss it with Dan together.